src/cluster.c

```diff
@@ -3306,7 +3378,7 @@ void clusterHandleSlaveFailover(void) {
         if (server.cluster->mf_end) {
             server.cluster->failover_auth_time = mstime();
             server.cluster->failover_auth_rank = 0;
-	    clusterDoBeforeSleep(CLUSTER_TODO_HANDLE_FAILOVER);
+	        clusterDoBeforeSleep(CLUSTER_TODO_HANDLE_FAILOVER);
```

Comment:
You tried to fix this, but there is actually a tab here which is causing the incorrect spacing.